src/main/io/serial_cli.c

```diff
@@ -120,6 +121,9 @@ static void cliServoMix(char *cmdline);
 static void cliSet(char *cmdline);
 static void cliGet(char *cmdline);
 static void cliStatus(char *cmdline);
+#ifndef SKIP_TASK_STATISTICS
```

Comment:
would prefer `USE_TASK_STATISTICS` then all the `#ifndef SKIP_...` gets replaced with a positive include `#ifdef USE_...`